### ISSUE TITLE
NameError uninitialized constant

### ISSUE THREAD

**Issue by ALYREZA** [jcs/lobsters#373]
every thing, every thing works in development perfect,
but when I deployed to server : 
* does not fetch title when i gonna create new story
* does not create new story , blank page appeared
* preview story does not work
* can not make post request , give it error 500
production.log 
```
 Completed 500 Internal Server Error in 30ms (ActiveRecord: 5.2ms)
 NameError (uninitialized constant #<Class:0x0056545841e008>::Markdowner):

 app/models/story.rb:352:in `generated_markeddown_description'
 app/models/story.rb:295:in `description='
 app/controllers/stories_controller.rb:15:in `create'
```
Ruby => 2.4.1
Rails => 5.1.1

**Comment by jcs** [comment]
That should have been fixed in 632b2b586309ae7253414fe9728f0287c8afeae2, are you up to date?

**Comment by ALYREZA** [comment]
now yessss
